R/interpret.R

```diff
@@ -157,3 +194,24 @@ plot.interpret <- function(x, include_intercept = FALSE, max_char = 40,
     print(the_plot)
   return(invisible(the_plot))
 }
+
+# Print interpret objects
+#' @export
+print.interpret <- function(x, ...) {
+  ref_levels <- attr(x, "ref_levels")
+  out <-
+    if (purrr::is_empty(ref_levels)) {
+      "There are no reference levels..."
+    } else {
+      paste0(c("Reference Levels:\n", paste0(
+        "All `", names(ref_levels), "` are relative to `", ref_levels, "`\n"
```

Comment:
Need an "estimates" here, if you're going to keep this line.

Reply:
Good catch. Thank you.